### PR TITLE
Added tests for scenario 25,26 in Auxiliary Image 4.0 test spec

### PR DESCRIPTION
**Pull request by marinakog**
see https://confluence.oraclecorp.com/confluence/display/WLSQA/Auxiliary+Image+4.0+Use+Cases

https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/9441

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java [hunk 1]
```diff
@@ -638,44 +647,181 @@ void testCreateDomainNoFilesAtSourceModelHome() {
...
+    assertNotNull(domainCR1, "failed to create domain resource");
+    domainCR1.spec().configuration().model().configMap(configMapName1);
+    createDomainAndVerify(domainUid, domainCR1, domainNamespace,
+            adminServerPodName, managedServerPrefix, replicaCount);
+  }
+
+  /**
+   * Create a domain using auxiliary image with custom wdtModelHome and wdtInstallHome
+   * where the wdtModelHome is placed
+   * under the wdtInstallHome directory, or same directory. Ditto for vice-versa.
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:789]: Describe the usecase clearly with out using the word 'ditto'  Give the example of the the parameter with the actual value used in the usecase

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java [hunk 2]
```diff
@@ -1173,23 +1323,93 @@ private Domain createDomainResourceWithAuxiliaryImage40(
       String auxiliaryImagePath,
       String... auxiliaryImageName) {
 
+    return createDomainResourceWithAuxiliaryImage40(
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java:1326]: Rename the method to createDomainResourceWithAuxiliaryImage()

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java [hunk 3]
```diff
@@ -754,6 +631,58 @@ public static Domain createDomainResourceWithAuxiliaryImage40(
     return domainCR;
   }
 
+  /**
+   * Create a domain object for a Kubernetes domain custom resource using the basic WLS image
+   * and MII auxiliary images containing the doamin or/and cluster configuration.
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/CommonMiiTestUtils.java:636]: typo doamin --> domain 
Do we support Auxiliary Image in cluster scope in Release 4.0 ?   This feature was in Release 3.x
In 4.0 we only support Aux images in  spec/configuration  scope. 

So the method name should not be createDomainResourceWithAuxiliaryImageClusterScope()

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java (hunk 1 above)
Comment (marinakog) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItMiiAuxiliaryImage.java]: done

**Review by anpanigr — APPROVED** [review]
lgtm

**Review by vanajamukkara — APPROVED** [review]
LGTM

**Pull request merged**